field/field.go

```diff
@@ -0,0 +1,140 @@
+package field
+
+import (
+	"fmt"
+	"github.com/wrthomps/GopherIt/move"
+	"strconv"
+	"strings"
+)
+
+const (
+	SEMI_COLON = ";"
+	COMMA      = ","
+
+	REPLACE_ALL = -1
+)
+
+type Field struct {
+	MyId       int
+	OpponentId int
+	Rows       int
+	Cols       int
+	field      [][]int
+}
+
+func (f *Field) Init() error {
+	if f.Rows <= 0 || f.Cols <= 0 {
+		return fmt.Errorf("Invalid settings for field, cannot initialize")
+	}
+
+	f.field = make([][]int, f.Cols)
+	for col := range f.field {
+		f.field[col] = make([]int, f.Rows)
+	}
+
+	f.clearField()
+	return nil
+}
+
+func (f *Field) ParseFromString(s string) {
+	replaced := strings.Replace(s, SEMI_COLON, COMMA, REPLACE_ALL)
+	r := strings.Split(replaced, COMMA)
+	counter := 0
+	for y := 0; y < f.Rows; y++ {
+		for x := 0; x < f.Cols; x++ {
+			f.field[x][y], _ = strconv.Atoi(r[counter])
+			counter++
+		}
+	}
+}
+
+func (f *Field) clearField() {
+	for y := 0; y < f.Rows; y++ {
+		for x := 0; x < f.Cols; x++ {
+			f.field[x][y] = 0
+		}
+	}
+}
+
+func (f *Field) AvailableMoves() []*move.Move {
```

Comment:
Doesn't handle ko or superko. Apparently superko isn't in the ruleset? Definitely handle ko, though.